apps/store/src/lib/l10n/locales.ts

```diff
@@ -60,7 +60,7 @@ export const normalizeLocale = (locale: string | undefined): string | undefined
 }
 
 // We use en-SE ISO format for settings but downcase it for routing to get nicer URLs
-export const routingLocale = (locale: string) => locale.toLowerCase()
+export const routingLocale = (locale: LocaleValues) => locale.toLowerCase() as RoutingLocale
```

Comment:
I'm no TS pro, but could this be or will TS complain?

```suggestion
export const routingLocale = (locale: LocaleValues):RoutingLocale => locale.toLowerCase()
```

Reply:
Yeahhh feels a lil dirty to be casting that type but TS complains without it since `toLowerCase()` only returns a `string`. So going in we have out clean and specific type. but coming out we only have a string :feelsbad:

Reply:
Can we worker around with conversion function that return proper type
`
asRoutingLocale = (locale: LocaleValues): RoutingLocale => {
  return locale.toLowerCase() as RoutingLocale
}
`

But only ways without assertion that I know are
- to have explicit mapping or explicit switch statement
- to check toLowerCase result agains all possible keys in RoutingLocale